reference/employer/openapi.yaml

```diff
@@ -0,0 +1,1083 @@
+                        account_name: CHECKING ACCOUNT
+                        institution_name: 'BANK OF AMERICA, N.A.'
+                        account_type: checking
+                        account_number: '98765432'
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                    description: A stable Finch `id` (UUID v4) for the company.
+                  legal_name:
+                    type: string
+                    description: The legal name of the company.
+                  entity:
+                    type: object
+                    description: The entity type object.
+                    properties:
+                      type:
+                        type: string
+                        description: The tax payer type of the company.
+                        enum:
+                          - llc
+                          - corporation
+                          - sole_proprietor
+                          - non_profit
+                          - partnership
+                          - cooperative
+                      subtype:
+                        type: string
+                        description: The tax payer subtype of the company.
+                        enum:
+                          - s_corporation
+                          - c_corporation
+                          - b_corporation
+                        nullable: true
+                  primary_email:
+                    type: string
+                    description: The email of the main administrator on the account.
+                    nullable: true
+                  primary_phone_number:
+                    type: string
+                    description: 'The phone number of the main administrator on the account. Format: `XXXXXXXXXX`'
+                    pattern: '[0-9]{10}'
+                    nullable: true
+                  departments:
+                    type: array
+                    description: The array of company departments.
+                    nullable: true
+                    items:
+                      type: object
+                      nullable: true
+                      properties:
+                        name:
+                          type: string
+                          description: The department name.
+                        parent:
+                          type: object
+                          description: 'The parent department, if present.'
+                          nullable: true
+                          properties:
+                            name:
+                              type: string
+                              description: The parent department's name.
+                  ein:
+                    type: string
+                    description: The employer identification number.
+                    nullable: true
+                  locations:
+                    type: array
```

Comment:
locations is nullable as well! (similar to accounts)

Reply:
this is a reference. the reference itself is nullable

Reply:
But the reference is an object..locations is an array of objects, right?